src/vs/workbench/services/workspaces/common/workspaceTrust.ts

```diff
@@ -418,4 +431,23 @@ export class WorkspaceTrustTransitionManager extends Disposable {
 	}
 }
 
+class WorkspaceTrustMemento {
```

Comment:
the title and usage of this memento aren't aligned. i understand with context that this is a memento for determining the trust state of empty workspaces, but if someone else comes in they might think this memento is how we determine workspace trust for all workspaces.

Reply:
I was thinking about that and for consistency I have removed the `_isWorkspaceTrusted` field and store the current workspace trust state in the memento as well. This way independent of the workspace type, the current workspace trust state is being stored consistently.